cypher-editor-support/src/completion/rules/ruleParameter.js

```diff
@@ -0,0 +1,11 @@
+import * as CompletionTypes from '../CompletionTypes';
```

Comment:
Name of the rule should be different. Parameters in general are handle in other rule.

This one is `ruleParameterIfDollarSignIsPresent.js` or something like that